webapp/src/app/shared/phaser/entities/registry/prop.ts

```diff
@@ -218,7 +218,67 @@ export class Prop extends CCEntity {
 			offset?: Point3;
 			renderMode?: string
 		}[] = [];
+
+
+		if (anims.sheet || anims.namedSheets) {
+			this.setupAnim(settings, anims, sheetDef, propDef.name, sprites);
+		} else { //<- temporary
```

Comment:
why temporary?

Reply:
Because some entities have a `sheet` as well as `SUB` entries and I think they are supposed to be rendered as if `sheet` was inserted as the first element into `SUB`.
Since `renderMode` has been fixed the else should be removed.

Reply:
Turns out that fixing that is a bit more complex and involves handeling `frame` entries. The current implementation should give a result that is good enough for an editor.